src/resources/vue/components/model.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <div class="bg-modal" @click="toggleModal" v-if="showModal">
+        <div v-if="showModal" class="modal" @click.stop>
+            <svg class="close-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" style="transform: rotate(45deg);">
```

Comment:
use Icon component

Reply:
and use remixicon.com, there is an icon called "close"